go.mod

```diff
@@ -99,3 +99,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.6.0 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/vmware-tanzu/velero => github.com/openshift/velero v0.10.2-0.20250701195919-c5900973e7ec
```

Comment:
Are we pulling in `oadp-dev` branch here ?
Also, we would need `oadp-dev` as default branch for this current repo too. (cc @mpryc )

Reply:
Yes, it is oadp-dev
Deterministic so if we want to move forward would need to set the branch and do another mod tidy